app/helpers/thredded/urls_helper.rb

```diff
@@ -58,5 +58,15 @@ def edit_post_path(post)
         edit_messageboard_topic_post_path(messageboard, post.postable, post)
       end
     end
+
+    # @param post [Post, PrivatePost]
+    # @return [String] path to the DELETE PATCH PUT POST endpoint.
+    def delete_post_path(post)
```

Comment:
@glebm you clobbered the `post_path` so I think it's behaving not as expected

```ruby
    # @param post [Post, PrivatePost]
    # @return [String] path to the topic page with the post anchor.
    def post_path(post, params = {})
      post_url(post, params.merge(only_path: true))
    end
```

resulting URL is (at least in the dummy app)

> http://localhost:9292/thredded/main-board/that-is-why-you-fail-eb7a11aa-27ca-4ea4-89f9-1c7331f05995/page-1#post_2101